source/docs/software/driverstation/driver-station-labview-dashboard.rst

```diff
@@ -0,0 +1,127 @@
+FRC Driver Station LabVIEW Dashboard
+====================================
+
+The Dashboard application installed and launched by the FRC Driver Station is a LabVIEW program designed to provide teams with basic feedback from their robot, with the ability to expand and customize the information to suit their needs. This Dashboard application uses Network Tables and contains a variety of tools that teams may find useful.
+
+LabVIEW Dashboard
+-----------------
+
+.. image:: images/driver-station-labview-dashboard/labview-dashboard.png
+
+The Dashboard is broken into two main sections. The left pane is for displaying a camera image. The right pane contains:
+
+- Drive tab that contains indicators for joystick and drive motor values (hooked up by default when used with LabVIEW robot code), a gyro indicator, an Autonomous selection text box, a connection indicator and some controls and indicators for the camera,
+- Basic tab that contains some default controls and indicators,
+- Custom tab for customizing the dashboard using LabVIEW
+- Test tab for use with Test Mode in the LabVIEW framework
```

Comment:
Add custom before Commands - placeholder for custom dashboards

Reply:
Does this sequence make sense?

```
The Dashboard is broken into two main sections. The left pane is for displaying a camera image. The right pane contains:

- Drive tab that contains indicators for joystick and drive motor values (hooked up by default when used with LabVIEW robot code), a gyro indicator, an Autonomous selection text box, a connection indicator and some controls and indicators for the camera
- Basic tab that contains some default controls and indicators
- Test tab for use with Test Mode in the LabVIEW framework
- Custom tab for customizing the dashboard using LabVIEW
- Commands tab for use with the new LabVIEW C&C Framework
- Checklist tab that can be used to create task lists to complete before and/or between matches
- Variables tab that displays the raw Network Tables variables in a tree view format

Reply:
Missing Camera tab between drive and basic.
The order should match the dashboard
drive, basic, camera, custom, test, commands, checklist, variables

Reply:
Fixed in 36286f9.